numpy/f2py/crackfortran.py

```diff
@@ -2985,7 +2983,6 @@ def expr2name(a, block, args=[]):
 
 def analyzeargs(block):
     setmesstext(block)
-    implicitrules, attrrules = buildimplicitrules(block)
```

Comment:
This too looks like a problem: why isn't the result of `buildimplicit` rules used?

Please remove from the PR for now.

Reply:
Here `attrrules` is not used. One could replace the statement with
```
implicitrules = buildimplicitrules(block)[0]
```
but removing the `buildimplicitrules` call would be wrong.